src/app/common/toast/epics.ts

```diff
@@ -2,44 +2,45 @@ import { Store } from 'redux'
 import { ActionsObservable } from 'redux-observable'
 import { Observable } from 'rxjs/Observable'
 
-import * as PaymentsActionNames from '../../payments/action-names'
-import { PaymentsActions } from '../../payments/types'
+import { ActionList as PaymentActionList, ActionType as PaymentsActionType } from '../../payments/types'
 import * as ToastActionNames from './action-names'
 import { FAILED, SEND_FAIL, SEND_SUCCESS, SUCCESS } from './toast-messages'
 import { ToastActions } from './types'
 
-const MAX_DELAY: number = 5000
+const MAX_DELAY = 5000
```

Comment:
Why was the type def removed?

Reply:
I rearranged the rules. Obvious types are not needed anymore. It will even trigger an error if you do.